lib/LaTeXML/Post/MathML/OperatorDictionary.pm

```diff
@@ -116,8 +116,8 @@ sub lookup_category {
     #   then replace Content with the Unicode character "U+0320
     #   plus the index of Content in Operators_2_ascii_chars"
     #   and move to step 3.
-    elsif (my ($p) = grep { $$Operators_2_ascii_chars[$_] ? ($_) : (); } 0 .. $#$Operators_2_ascii_chars) {
-      $code1 += $p;
+    elsif (my $p = $$Operators_2_ascii_chars{$content}) {
+      $code1   = 0x320 + $p;
```

Comment:
Subtle edge case detail: 

```perl
elsif (my ($p) = (0)) {...}
``` 
from the previous code would have evaluated as true. 

But the hash lookup approach would be false for the 0 index. Probably gfood to add a test for `!!`.

Reply:
Can we just hardcode the exact final character codes in `$Operators_2_ascii_chars` and avoid the runtime arithmetic? Should be safer (and won't have 0 values).

Reply:
Good catch! Now harcoded directly in the hash.